src/QuantumToolbox.jl

```diff
@@ -35,6 +35,7 @@ import SciMLBase:
 import StochasticDiffEq: StochasticDiffEqAlgorithm, SRA1
 import SciMLOperators:
     AbstractSciMLOperator,
+    AddedOperator,
```

Comment:
Why do we need to import this operator? I think we are not using it

Reply:
ohh yes,
sorry, forgot to remove it